test/other/test_emit_tsd_wasm_only.d.ts

```diff
@@ -0,0 +1,8 @@
+// TypeScript bindings for emscripten-generated code.  Automatically generated at compile time.
+interface WasmModule {
+  _fooVoid(): void;
+  _fooInt(_0: number, _1: number): number;
+  __start(): void;
+}
```

Comment:
Is this actually useful though?

Can use this typescript to load the wasm file?  Maybe it would better to just error in this case?

Reply:
If the user was creating their own loader, they could use the definitions.

Reply:
Uhg...but these will be the mangled named with the `_` which won't match the actual exports.

Reply:
if you think its worth it for keep this thats fine with me.  lgtm either way.

Reply:
We should try to get rid of that mangling at some point ... but that is a much bigger issue.

Reply:
I don't think it's worth it with the name mangling. I'll close this and add an error.